call.go

```diff
@@ -143,6 +166,9 @@ func Invoke(ctx context.Context, method string, args, reply interface{}, cc *Cli
 			}
 			return toRPCErr(err)
 		}
+		if enableTracing {
+			c.traceInfo.tr.LazyLog(payload{args}, true)
```

Comment:
`/debug/requests` only responds to requests from localhost by default, so it's somewhat secure in that regard. Programs with different requirements (whether more or less stringent) can control that through the trace package.

This will pin payloads (out of necessity) for a certain length of time, but there's a bounded number that will be stored.